package/scripts/setup-canvaskit.js

```diff
@@ -61,9 +88,48 @@ function copyFile(from, to) {
   fs.writeFileSync(to, data);
 }
 
-// Copy the WASM file to `<static>/static/js/canvaskit.wasm`
+function resolveFsAndPath() {
+  try {
+    const canvasKitPackageJsonPath = path.join(process.cwd(), 'node_modules', /* '@shopify', 'react-native-skia', 'node_modules', */ 'canvaskit-wasm', 'package.json');
+
+    console.log(
+      `› Optionnal metro bundler step:\n  ${gray(`Loading 'canvaskit-wasm' package.json from:\n  ${canvasKitPackageJsonPath}`)}`
+    );
+
+    const canvasKitPackageJson = require(canvasKitPackageJsonPath);
+    // eslint-disable-next-line import/no-dynamic-require
+    canvasKitPackageJson.browser = {
+      fs: false,
+      path: false,
+      os: false,
+    };
+    
+    console.log(`  ${gray(`Saving the updated 'canvaskit-wasm' package.json to resolve 'fs' and 'path' modules.`)}\n`);
+    
+    fs.writeFileSync(
+      canvasKitPackageJsonPath,
+      JSON.stringify(canvasKitPackageJson, null, 2)
+    );
+  } catch (error) {
+    // console.error(
+    //   `Could not update 'canvaskit-wasm' package.json to disable 'fs' and 'path' modules. Please install '@shopify/react-native-skia' and ensure it can be resolved from your project: ${process.cwd()}`
+    // );
+    console.log(error);
+    process.exit(1);
+  }
+}
+
 (() => {
-  copyFile(getWasmFilePath(), getOutputFilePath());
+  // Automatically detect if it's an expo project with a metro bundler
+  const isAnExpoProjectWithMetro = getWetherItsAnExpoProjectWithMetro();
+
+  // Copy the WASM file to `<static>/static/js/canvaskit.wasm`
+  copyFile(getWasmFilePath(), getOutputFilePath(isAnExpoProjectWithMetro));
```

Comment:
shouldn't we copy it to /public/canvaskit.wasm if `isAnExpoProjectWithMetro`? Maybe not actually. No sure.

Reply:
would it make sense to use `/public/canvaskit.wasm`? the `/public/static/js/canvaskit.wasm` path is really a legacy of current expo web projects being stucked at webpack v4

Reply:
Ah yes interesting, didn't have this historical context.

Folks might have other files in their public folder, so even though the `static/` folder is overkill, it might be good to keep the `js/` one so that it's clear that is a JS script related file. 

👉 So `/public/js/canvaskit.wasm`, what do you think?

Reply:
Well, I'm currently managing another project with a WASM file and it does need to be at the root for the `emscripten` compiled code to load it dynamically. But I hate it. I would love to be able to have it in a separate folder like `js`, and I will eventually rewrite the compiled code so it can be done.

Good news: with RN Skia, we can tell the script where to get the .wasm file is (through `LoadSkiaWeb` and `WithSkiaWeb`), so no added value of having it at the root... 

Let's keep things nice and tidy and have it under a `js/` folder so it doesn't mess too much with people's `public/` folder! ✨

I will update the script and the docs accordingly